lib/components/atoms/Blockquote/Blockquote.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+/**
+ *
+ * Blockquote
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import styles from './Blockquote.style';
+import type { Props } from './types';
+
+const Blockquote = ({ className, quoteProp, quoteAuthorProp }: Props): Node => (
```

Comment:
Why this component is part of your PR @ritugupta64 ?

Reply:
fixed